### DISCUSSION TITLE
found a possible phishing site that's a clone of github

### DISCUSSION THREAD

**Discussion by ingenarel** [community/community#140082] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

i was searching for some neovim plugins via duckduckgo, then i clicked a link, `https://hub.apw.app` and it looks a replica of github.

i reported the link in google's safebrowsing and to microsoft. if it's an alt link for github, then it won't hurt. and if it's probably a phishing link, then win ig.

**Comment by AidenJamesYt** [comment]
if anything, open a support ticket, and provide the link

**Comment by LiteBrite82** [comment, marked as answer]
@ingenarel 

Great find! ❤️ 🚀 🎆  Our security team has taken action

Thank you so much for taking action to keep the community, and all of our users safe!!
